test/webserv/integration/test_connection.py

```
@@ -153,3 +166,105 @@
+
+        # GET JSONしてContent-Typeがjsonかどうかをassert
+        con.request("GET", "/" + UPLOAD_DIR + JSON_FILENAME)
+        response = con.getresponse()
+
+        assert_status_line(response, HTTPStatus.OK)
+        assert_header(response, CONTENT_TYPE, APPLICATION_JSON)
+        assert response.read() == json.dumps(JSON_DATA).encode("utf-8")
+
+    except HTTPException as e:
+        print(f"Request failed: {e}")
+        raise AssertionError
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)